src/com/encora/javaAssignments/DailyProgramPractice/program7/SearchGivenElement.java

```diff
@@ -0,0 +1,22 @@
+package com.encora.javaAssignments.DailyProgramPractice.program7;
+
+public class SearchGivenElement {
+    public static int search(int array[], int firstNumber, int secondNumber) {
```

Comment:
Give proper variable name for(firstNumber and secondNumber)

Reply:
Done